test/unit/vdom/skip.js

```diff
@@ -10,6 +10,7 @@ describe('vdom/skip', () => {
       },
       render() {
         vdom.element('div', { skip: true }, () => {
+          vdom.text('text');
```

Comment:
@nadiam84 I've updated the tests. Can you pls have a look when you have a chance? Thanks!